test-integration/sonar-pr/docker-compose.yml

```
@@ -0,0 +1,13 @@
+services:
+  sonarqube:
+    image: mc1arke/sonarqube-with-community-branch-plugin:10.3-community
+    container_name: sonar-server
+    environment:
+      - SONAR_ES_BOOTSTRAP_CHECKS_DISABLE=true
+    ports:
+      - '9000:9000'
+    healthcheck:
```

We should download the branch plugin and put in ./extension folder.
```suggestion
    volumes:
      - ./extensions:/opt/sonarqube/extensions
    healthcheck:
```

---

I think thats a great idea, but how do we keep versions compatible in case of renovate updates?

---

Renovate with custom-manager and github-releases.